src/nft-card-front.ts

```diff
@@ -202,24 +222,15 @@ export class NftCardFrontTemplate extends LitElement {
   }
 
   public getAssetPriceTemplate() {
-    // TODO: Needs to account for tokens with images not symbols
-    // If payment_token.image_url then use token image instead of symbol
-    let prevPriceTemplate: TemplateResult = html``
-    let currentPriceTemplate: TemplateResult = html``
-
-    if (this.lastSaleData?.paymentToken) {
-      // const currentPriceSymbol = this.lastSaleData.paymentToken.symbol === 'ETH' ? 'Ξ ' : ''
-      currentPriceTemplate = this.getCurrentPriceTemplate(this.lastSaleData.paymentToken, this.lastSaleData.currentPrice)
 
-    }
+    const currentPriceTemplate = this.lastSaleData?.paymentToken ?
+        NftCardFrontTemplate.getPriceTemplate(PriceType.Current, this.lastSaleData.paymentToken,
+            this.lastSaleData.currentPrice) : null
```

Comment:
why is the "currentPrice" under a variable called `lastSaleData`? that's very confusing. we have `this.asset` in this component right? the top asset.sellOrder would have the `currentPrice`